src/components/ExploreMovieCardBottom.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Movie, MoviePreference, PreferenceType } from "../types";
+
+import { Button, Icon } from "semantic-ui-react";
+
+const ExploreMovieCardBottom = (props: {
```

Comment:
Do you think we'll reuse this component somewhere else? It could make sense to name it something more targeted around what it does instead of where it goes.